src/Sylius/Bundle/CoreBundle/Resources/translations/validators.en.yml

```diff
@@ -96,6 +96,7 @@ sylius:
         currency_code:
             not_valid: The currency code you entered is invalid.
         invalid_state_transition: 'Cannot complete as order is in a wrong state. Current: %currentState%. Possible transitions: %possibleTransitions%.'
+        address_requirement: 'Please provide %addressName%.'
```

Comment:
Split it into two different messages.
`Please provide a shipping address`
`Please provide a billing address`

Reply:
Ok, I will fix this in next PR